src/main/webapp/webpack.config.js

```diff
@@ -1,128 +1,103 @@
 const path = require("path");
```

Comment:
Please add some comments to this file as well

Reply:
Updated in 622d87c